.github/workflows/update-wpt.yml

```
@@ -0,0 +1,52 @@
+name: Update WPT
+
+on:
+  workflow_dispatch:
+  schedule:
+    # Run once a week at 12:00 AM UTC on Sunday.
+    - cron: 0 0 * * *
+
+jobs:
+  update-wpt:
+    name: Update WPT
+    permissions:
+      contents: write
+      pull-requests: write
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@692973e3d937129bcbf40652eb9f2f61becf3332  # v4.1.7
+        with:
+          persist-credentials: false
+      - name: Update WPT
+        run: |
+          rm -rf test/fixtures/wpt && mkdir test/fixtures/wpt &&
+      - name: Clone WPT Repository
+        uses: actions/checkout@v2
+        with:
+          repository: web-platform-tests/wpt
+          ref: ${{ github.sha }}
+          path: test/fixtures/tmp-wpt
+          sparse_checkout: common/ eventsource/ fetch/ interfaces/ mimesniff/ resources/ service-workers/ storage/ websockets/ xhr/
```

```suggestion
          sparse_checkout: common/ eventsource/ fetch/ interfaces/ mimesniff/ resources/ service-workers/ storage/ websockets/ xhr/ LICENSE.md
```

Also this can go on separate lines, right?